Extensions/PlatformBehavior/platformtools.ts

```diff
@@ -0,0 +1,18 @@
+namespace gdjs {
+  export namespace platform {
+    export const isOnPlatform = function (
+      objectsLists1,
+      behavior,
+      objectsLists2,
+      inverted
```

Comment:
Let's add type for these. 
objectLists are `Hashtable<Array<gdjs.RuntimeObject>>` 
behavior is a string, can you rename it to behaviorName?